feat: Added skipLocalhost option (#17)
davidtheclark/remark-lint-no-dead-urls: README.md

- **skipOffline** `{boolean}` - Default: `false`.
  By default, if you are offline when you run the check you will receive a warning.
  If you want to let offline runs quietly pass, set this option to `true`.
- **skipLocalhost** `{boolean}` - Default: `false`.

>   By default, all local links testing to a live status, and if project not running locally you'll receive a warning.

Reviewer: ```suggestion
  By default, `localhost` links are treated the same as other links, so if your project is not funning locally you'll receive a warning.
```

Author: _funning_ should be _running_, right?

>   If you want to skip localhost links (http://localhost, http://127.0.0.1), set this option to `true`.

Reviewer: ```suggestion
  If you want to ignore `localhost` links (e.g. `http://localhost/*`, `http://127.0.0.1/*`), set this option to `true`.
```
